entity-service/build.gradle.kts

```diff
@@ -22,7 +22,7 @@ tasks.integrationTest {
 dependencies {
   implementation(project(":entity-service-factory"))
 
-  implementation("org.hypertrace.core.serviceframework:platform-grpc-service-framework:0.1.49")
+  implementation("org.hypertrace.core.serviceframework:platform-grpc-service-framework:0.1.52")
   implementation("org.hypertrace.core.documentstore:document-store:0.7.29")
 
   runtimeOnly("io.grpc:grpc-netty:1.50.0")
```

Comment:
especially this one - mismatching the grpc runtime (netty) with the grpc libs will cause a runtime failure.